### ISSUE TITLE
setup scripts complain about /bin/sh being symlinked to dash instead of bash

### ISSUE THREAD

**Issue by sneak** [Angstrom-distribution/setup-scripts#3]
The scripts already detect dash instead of bash.  With this condition they could simply use /bin/bash instead of /bin/sh.

Suggesting that I reconfigure my /bin in the case of a site-specific configuration _already detected_ by the package is rude.

**Comment by SamB** [comment]
Yes, while the current state of affairs is certainly better than failing somewhere in the middle in a difficult-to-diagnose fashion, it would be better to make your scripts work with /bin/sh -> dash ...

**Comment by koenkooi** [comment]
@SamB it's not the scripts under angstrom control, it's the buildscripts of the stuff we build that still aren't dash safe. >99% is, but I still hit a few issues every now and then :(
Having said that, if the scripts that are under angstrom control are broken under dash we should fix that.
